Containers/mastercontainer/cron.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
-set -eu
+set -eux
 
 while true; do
     sudo -u www-data php /var/www/docker-aio/php/src/Cron/cron.php
+    sudo -u www-data docker image prune -f
+    sudo -u www-data docker volume prune -f
```

Comment:
Could there be scenarios where the cron is running while a container is recreated, so volumes might not be attached to any container during that time? I'd consider that a bit dangerous then to prune the volumes with potential data on it.

Reply:
Thats indeed possible:
https://github.com/nextcloud/all-in-one/blob/c203ad65ef32531ae700a9b2cfce6fcf7d7103ee/php/src/Controller/DockerController.php#L36-L39
Do you have any other idea how we could handle this? Without running the volume prune regularly, there will be a lot of unnecessary volumes after a while...

Reply:
Is there a separation between volumes that should be persistent and ones that would be only temporary per container? Maybe if all of the persistent ones are named, then they should not change during recreation and you could filter them out of a `docker volume ls` and remove the remaining ones individually.

Reply:
The thing is we don't know if the remaining ones without ours are stale. At least I don't know how to get this info without doing a docker inspect.
Perfect would be a dry-run command that we could use to get all the stale volumes and then just filter out ours and remove the rest. Unfortunately that doesnt exist yet:
https://github.com/moby/moby/issues/30623

Reply:
Okay, seems like we can simply run a docker image ls --filter dangling=true

Reply:
should be fixed now :)